crypto.pl

```diff
@@ -33,7 +33,12 @@
 */
 
 :- module(crypto,
-          [ evp_decrypt/6,              % +CipherText, +Algorithm, +Key, +IV, -PlainText, +Options
+          [ crypto_term_hash/3,         % +Data, -Hash, +Options
```

Comment:
The name is potentially confusing with the widely accepted term_hash/2,4 family, while _data_ can only be text.  crypto_string_hash/3?